README.md

```diff
@@ -988,14 +989,16 @@ main:
       limit: 10
 ```
 
-> [!TIP]
+> [!TIP] 
+
 > **Example cookbook**: [cookbooks/10-Document_Management_and_Search.py](https://github.com/julep-ai/julep/blob/dev/cookbooks/10-Document_Management_and_Search.py)
 
 ### Built-in `integrations`
 
 Julep comes with a number of built-in integrations (as described in the section below). `integration` tools are directly executed on the julep backend. Any additional parameters needed by them at runtime can be set in the agent/session/user's `metadata` fields.
 
-> [!TIP]
+> [!TIP] 
+
```

Comment:
remove this; it breaks github special syntax

Reply:
@creatorrr , have correct all these syntaxes